content_cafe.py

```diff
@@ -55,6 +55,7 @@ def process_item(self, identifier):
             self.content_cafe.mirror_resources(identifier)
             return identifier
         except Exception as e:
+            self.log.error('Coverage error for %r', identifier, exc_info=e)
```

Comment:
There's a recurring AttributeError on alpha, and I'm hoping logging the stacktrace will provide more information.